e2e/Sandbox/DashboardCreators/WebServiceDataSourceDashboard.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Reveal.Sdk.Data.MySql;
+using Reveal.Sdk.Data.Rest;
+using Reveal.Sdk.Dom;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Sandbox.DashboardFactories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sandbox.DashboardCreators
+{
+    public class WebServiceDataSourceDashboard : IDashboardCreator
+    {
+        public string Name => "Web Service Data Source";
+
+        public RdashDocument CreateDashboard()
+        {
+            var dataSource = new WebServiceDataSource()
```

Comment:
we want to simplify the use of this so that the WebServiceDataSource object i snot needed. Look at the RestDataSourceItem as an example.